include/ImmediateContext.hpp

```diff
@@ -1421,6 +1421,16 @@ class ImmediateContext
     void RollOverHeap(OnlineDescriptorHeap& Heap) noexcept(false);
     UINT ReserveSlotsForBindings(OnlineDescriptorHeap& Heap, UINT (ImmediateContext::*pfnCalcRequiredSlots)()) noexcept(false);
     UINT ReserveSlots(OnlineDescriptorHeap& Heap, UINT NumSlots) noexcept(false);
+    /// <summary>
+    /// Copy descriptors referenced by the handles in pSrcDescriptorHandles to a linear set of HeapSlots starting at HeapSlot.
+    /// Note that both HeapSlot and currentStateGpuDescriptorHandle are updated by this method.

```

Comment:
Stale reference to another parameter that wasn't used?

Reply:
Oh yep, looks that way. I'll fix that up